modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"dependencies": {
+		"metal-component": "^2.14.0",
+		"metal-soy": "^2.14.0"
+	},
+	"devDependencies": {
+		"babel-cli": "6.26.0",
```

Comment:
Are all these dependencies locked down and will never more or float up?

Reply:
At a first moment, I don't believe it'll change for 7.1, but I'll raise this question to the Forms team.

Reply:
Hey @gamerson, the versions are not supposed to change.